### DISCUSSION TITLE
Goerli Update | Aave & Full-Stack Update

### DISCUSSION THREAD

**Discussion by PatrickAlphaC** [smartcontractkit/full-blockchain-solidity-course-py#1702] [Announcements]
Hi all!

With the merge, we are sad to say that Kovan and Rinkeby are being sunset at some point, and a lot of the tooling is already starting to drop support. So we are moving to Goerli for the course!

Additionally, due to this, Lessons 13 & 10 are a little... wonky. So I've archived those repos for now.

If you want to do more front-end work, the new javascript/hardhat github/course is fantastic! 
https://github.com/smartcontractkit/full-blockchain-solidity-course-js

You can look for the full-stack courses to get a better understanding. For the Aave stuff, it's recommended that you work with the mainnet-fork as opposed to deploying to a testnet. OR if you really want, you can use the V3 aave contracts after a little digging yourself!

Happy learning all!

**Comment by PatrickAlphaC** [comment]
Also, if people want to double check my goerli changes didn't break anything, that would be much appreciated :)

**Comment by coodered** [comment]
Very thank for your job Patrick
